content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+   {{</* /note */>}}
+
+1. A third item in a list
+
+1. A fourth item in a list
+```
+
+The output is:
+
+1. Use the note shortcode in a list
+
+1. A second item with an embedded note
+
+    {{< note >}}
+    Warning, Caution, and Note shortcodes, embedded in lists, need to be indented four spaces. See [Common Shortcode Issues](#common-shortcode-issues).
+    {{< /note >}}
+
+1. A third item in a list
+
+1. A fourth item in a list
+
+### Caution
+
+Use `{{</* caution */>}}` to call attention to an important piece of information to avoid pitfalls.
+
+For example:
+
+```go-html-template
+{{</* caution */>}}
+The callout style only applies to the line directly above the tag.
+{{</* /caution */>}}
+```
+
+The output is:
+
+{{< caution >}}
+The callout style only applies to the line directly above the tag.
+{{< /caution >}}
+
+### Warning
+
+Use `{{</* warning */>}}` to indicate danger or a piece of information that is crucial to follow.
+
+For example:
+
+```go-html-template
+{{</* warning */>}}
+Beware.
+{{</* /warning */>}}
+```
+
+The output is:
+
+{{< warning >}}
+Beware.
+{{< /warning >}}
+
+## Markdown elements
+
+### Line breaks
+
+Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. Second-level headings follow the first-level (or the title) without any preceding paragraphs or texts. A two line spacing helps visualize the overall structure of content in a code editor better.
+
+### Headings
+
+People accessing this documentation may use a screen reader or other assistive technology (AT). [Screen readers](https://en.wikipedia.org/wiki/Screen_reader) are linear output devices, they output items on a page one at a time. If there is a lot of content on a page, you can use headings to give the page an internal structure. A good page structure helps all readers to easily navigate the page or filter topics of interest.
+
+Do | Don't
+:--| :-----
+Update the title in the front matter of the page or blog post. | Use first level heading, as Hugo automatically converts the title in the front matter of the page into a first-level heading.
+Use ordered headings to provide a meaningful high-level outline of your content. | Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.
+Use pound or hash signs (`#`) for non-blog post content. | Use underlines (`---` or `===`) to designate first-level headings.
+Use sentence case for headings. For example, **Clone the git repository** | Use title case for headings. For example, **Clone the Git Repository**
+
+### Paragraphs
+
+Do | Don't
+:--| :-----
+Try to keep paragraphs under 6 sentences. | Indent the first paragraph with space characters. For example, ⋅⋅⋅Three spaces before a paragraph will indent it.
```

Comment:
These do/don't pairs are usually on the same topic, but this pair is about different things. Perhaps take the advice out of the table.